repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/macro/sections/sections.ftl

```diff
@@ -3,6 +3,9 @@
 <#include "imageSection.ftl">
 <#include "related-linkSection.ftl">
 
+<!-- Set up equation support from mathjax -->
+<script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/latest.js?config=TeX-AMS_HTML"></script>
```

Comment:
Same question as above, regarding storing the file within our codebase and serving it from our instances of Hippo.